### PR TITLE
change default ttl to INFINITE

### PR DESCRIPTION
**Pull request by YiMysty**
Please have a look at our [guidelines for contributions](https://github.com/microsoft/appcenter-sdk-android/blob/develop/CONTRIBUTING.md) and consider the following before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.

## Description

https://dev.azure.com/msmobilecenter/Mobile-Center/_workitems/edit/63664

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

**Comment by YiMysty** [comment]
@guperrot is there a document about how to do that? 🤔 not quite know how to do it~

**Comment by YiMysty** [comment]
oh, the md file! got it!

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * **[Fix]** Remove unsecure UUID fallback when UUID generation theorically fails, in reality it never fails.
 * **[Fix]** Check for running in App Center Test will now work when using AndroidX instead of the support library.
 * **[Feature]** Add `AppCenter.isRunningInAppCenterTestCloud` to provide method to check if the application is running in Test Cloud.
+* **[Feature]** Change the default offline persistence from 1 day to infinite.
```
Comment (guperrot) [inline CHANGELOG.md]: Please look at previous versions, this should be in App Center Data section.

**Comment by YiMysty** [comment]
@dhei thanks~ searched keyword day in data folder, that should be all!

File: CHANGELOG.md (hunk 1 above)
Comment (YiMysty) [reply CHANGELOG.md]: oh oh! checking...

Comment (YiMysty) [reply CHANGELOG.md]: done~

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@
 * **[Fix]** Check for running in App Center Test will now work when using AndroidX instead of the support library.
 * **[Feature]** Add `AppCenter.isRunningInAppCenterTestCloud` to provide method to check if the application is running in Test Cloud.
 
+### AppCenterData
```
Comment (guperrot) [inline CHANGELOG.md]:
```suggestion
### App Center Data
```
We are space friendly in recent versions.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,10 @@
 * **[Fix]** Check for running in App Center Test will now work when using AndroidX instead of the support library.
 * **[Feature]** Add `AppCenter.isRunningInAppCenterTestCloud` to provide method to check if the application is running in Test Cloud.
 
+### App Center Data
+
+* **[Feature]** Change the default offline persistence from 1 day to infinite.
```
Comment (dhei) [inline CHANGELOG.md]: Let's use the same wording as https://github.com/microsoft/appcenter-sdk-apple/pull/1678 for consistency.

File: sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/TimeToLive.java [hunk 4]
```diff
@@ -20,7 +20,7 @@
     public static final int NO_CACHE = 0;
 
     /**
-     * Default caching value of one day (in seconds).
+     * Default caching value is cache does not expire.
```
Comment (dhei) [inline sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/TimeToLive.java]:
```suggestion
     * Default caching value.
```

File: sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/models/BaseOptions.java [hunk 5]
```diff
@@ -23,7 +23,7 @@
     }
 
     /**
-     * @return document time-to-live in seconds (default to one day).
+     * @return document time-to-live in seconds (default to infinite).
```
Comment (dhei) [inline sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/models/BaseOptions.java]:
```suggestion
     * @return document time-to-live in seconds.
```

**Pull request merged**